mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -208,13 +208,15 @@ def get_event(self, validated_data):
         if old_version:
             return (event_type, None)
         else:
-            event = (event_type, event_type_reason)
+            event = (
+                (event_type, event_type_reason) if event_type_reason else (event_type,)
+            )
             if event not in provider_mapping.AGENCY_EVENT_TO_PROVIDER_REASON.keys():
```

Comment:
Not related but when I moved away the private API code, pylint made me realize this `.keys()` is not required. `dict` already knows how to do a lookup on its keys.